docs/netdevsim-ptp-simulation.md

```
@@ -0,0 +1,595 @@
+---
+
+## 2. Kernel: netdevsim DPLL/GNSS/UBX
+
+**File:** `drivers/net/netdevsim/dpll.c`
+
+### 2.1 DPLL Emulation
+
+A pair of DPLL devices is registered when a netdevsim device is created
+with `wpc=1`:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)